nixos/lib/make-options-doc/mergeJSON.py

```diff
@@ -41,6 +41,49 @@ def unpivot(options: Dict[Key, Option]) -> Dict[str, JSON]:
         result[opt.name] = opt.value
     return result
 
+# converts in-place!
+def convertMD(options: Dict[str, Any]) -> str:
+    import mistune
+    from xml.sax.saxutils import escape
+
+    def apply(node) -> str:
+        if node['type'] == 'paragraph':
+            return "".join(map(apply, node['children']))
+        elif node['type'] == 'text':
+            return escape(node['text'])
+        elif node['type'] == 'codespan':
+            return f"<literal>{escape(node['text'])}</literal>"
+        elif node['type'] == 'block_code':
+            return f"<programlisting>{escape(node['text'])}</programlisting>"
```

Comment:
Would be nice to include the [`language` attribute](https://tdg.docbook.org/tdg/5.1/programlisting.html#element.db.programlisting.5.4).